packages/shield-controller/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.1.1]
```

Comment:
It looks like you added something to the API. Should this be 2.2.0 instead of 2.1.1?